src/distributed/cuda/nccl_communicator.cc

```diff
@@ -8,68 +8,140 @@
  * \brief NCCL Communicator.
  */
 
+#include <numeric>
+#include "raf/mpi_communicator.h"
 #include "raf/nccl_communicator.h"
+#include "./nccl_utils.h"
 
 namespace raf {
 namespace distributed {
 namespace communicator {
 
+#define NCCL_UNIQUE_ID_BYTES 128
+
+class NCCLIdSyncHelper {
+ public:
+  NCCLIdSyncHelper() {
+    const char* temp = getenv("RAF_FILE_STORE_PATH");
+    if (temp == nullptr) {
+      LOG(FATAL) << "RAF_FILE_STORE_PATH is no set.";
```

Comment:
Would that be better if we use a default path (e.g., `pwd`, `/tmp`) and just throw a warning?

Reply:
I endorse. It's better to set a default path.  Just want to double check, we are using `RAF_FILE_STORE_PATH` if we use `void` communicator.